Added new extra - avatar

SaijyotiTripathy opened the pull request:
Hi @arnaudmiribel ,

I am new to contributing to GitHub and raising pull requests. I have added a new Streamlit extra to your package and followed the contributing guidelines accordingly. I have also raised a PR for this addition.
Could you please guide me on the next steps? Here is how the extra looks like:
![Screenshot from 2025-03-11 14-01-22](https://github.com/user-attachments/assets/dc29bbbc-e088-467a-8dd1-1baa05eaf438)

Looking forward to your response.

Thank you!

arnaudmiribel commented on src/streamlit_extras/avatar/__init__.py:
```diff
@@ -0,0 +1,33 @@
+import base64
```
I don't think that's used? Can drop it

SaijyotiTripathy commented:
Thank you for the guidance! I have fixed the issues, removed the unused import, ran pre-commit, updated the lock file, and raised the PR again. Please let me know if any further changes are needed.
